rest-api-spec/src/main/resources/rest-api-spec/test/search/350_binary_field.yml

```diff
@@ -0,0 +1,49 @@
+---
+"binary":
+    - skip:
+        features: ["headers"]
+        version: " - 7.99.99"
+        reason: "docvalues_fields on binary field were corrected in 8.0"
+    - do:
+        indices.create:
+            index: test
+            body:
+                mappings:
+                    properties:
+                        binary:
+                            type: binary
+                            doc_values: true
+
+    - do:
+        #other formats (e.g. cbor) may not support parsing of binary
+        headers:
+            Content-Type: application/json
+        index:
+            index: test
+            id: 1
+            body:
+                binary: U29tZSBiaW5hcnkgYmxvYg==
+
+    - do:
```

Comment:
I'd stick the `refresh` as a parameter on `index`. Not a big thing, but I figure you may as well save a request.

Reply:
Addressed in 73be1d918a4e58be64be6dc4764091a090305d6c